core/src/main/java/org/mule/runtime/core/internal/store/PartitionedPersistentObjectStore.java

```diff
@@ -35,7 +35,7 @@
   private static final Logger LOGGER = getLogger(PartitionedPersistentObjectStore.class);
   public static final String OBJECT_STORE_DIR = "objectstore";
 
-  private MuleContext muleContext;
+  protected MuleContext muleContext;
```

Comment:
why?

Reply:
Part of PoC, not needed this change. Good catch!